ingestion/src/metadata/ingestion/source/database/salesforce/metadata.py

```diff
@@ -191,6 +193,26 @@ def get_tables_name_and_type(self) -> Optional[Iterable[Tuple[str, str]]]:
                 )
             )
 
+    def set_table_desc_map(self) -> None:
+        """
+        Method to set table description obtained gotten Tooling API to map
+        """
+        try:
+            results = self.client.toolingexecute(
+                "query/?q=SELECT+QualifiedApiName,Description+FROM+EntityDefinition"
+            )["records"]
```

Comment:
is `records` key expected to here in the response?

Reply:
@TeddyCr Yes, if the query is successful, we will get a response like [this](https://github.com/open-metadata/OpenMetadata/issues/13556#issuecomment-1769791063).

Reply:
I have handled the case of no key by using except block.